ledger/src/blockstore_meta.rs

```diff
@@ -214,6 +241,25 @@ impl SlotMeta {
         Some(self.consumed) == self.last_index.map(|ix| ix + 1)
     }
 
+    pub fn is_connected(&self) -> bool {
+        self.connected_flags.contains(ConnectedFlags::CONNECTED)
+    }
+
+    pub fn is_parent_connected(&self) -> bool {
+        self.connected_flags
+            .contains(ConnectedFlags::PARENT_CONNECTED)
+    }
+
+    pub fn set_connected(&mut self) {
```

Comment:
Maybe we should simply set two bits to true instead and have another function that only set the `01` bit to true?

Reply:
I think things might have changed slightly so marking this resolved; but please bring up again if you disagree.